modules/loans/src/lib.rs

```diff
@@ -268,7 +258,7 @@ impl<T: Config> Pallet<T> {
 	}
 
 	/// mutate records of collaterals and debits
-	fn update_loan(
+	pub fn update_loan(
 		who: &T::AccountId,
 		currency_id: CurrencyId,
 		collateral_adjustment: Amount,
```

Comment:
I don't like that I made this public, because I see why it was private as it is dangerous to use... but it was useful to use in the UpdateLoan trait